front/src/reducers/osrdconf/types.ts

```diff
@@ -67,6 +68,7 @@ export type OsrdStdcmConfState = OsrdConfState & {
   totalMass?: number;
   totalLength?: number;
   maxSpeed?: number;
+  loadingGauge?: LoadingGaugeType;
```

Comment:
Should we still use this editoast type if we just want to accept `GA` and `GB` ?
Also loading gauge we always be defined since it's selected to `GA` by default.
```suggestion
  loadingGauge: ['GA', 'GB'] as const;
```

We could also extract this a new type : 
```
type STDCMLoadingGauge = 'GA' | 'GB';
```

If you do, we can update the `const` in `StdcmConsist` with `const GAUGE_LIST: STDCMLoadingGauge[] = ...`

Reply:
Is it still relevant to do this with the other modification ?
It can be easier to add other GaugeType (G1, G2, GB1, ...) in the future with the LoadingGaugeType. I think that there's no way anything else that GA and GB can be stored by mistake (only stored from the Select that can select only GA and GB).

If it still relevant, i can add this type (in the types.ts file of src/applications/stdcm ?)

Reply:
No since the last changes, it's perfectly fine as it is, we can leave it that way :)